### PR TITLE
Core Testing: Wed 5 Jul 23

### PR DESCRIPTION
**Pull request by wanda-carlson**
Core Testing: Wed 5 Jul 23

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Hi **Slack user**, check this out.

**Comment by henry-richard** [comment]
this is a GH comment

File: Readme.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # public repo
```
Comment (henry-richard) [inline Readme.md:1]: do we need this?

File: Readme.md [hunk 2]
```diff
@@ -1,3 +1,2 @@
...
-2
-3
+> Core Testing: Wed 5 Jul 23, 4:38PM
```
Comment (henry-richard) [inline Readme.md:2]: we should add the timezone

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by henry-richard — APPROVED** [review]

**Pull request merged**